src/proof/arith_proof.cpp

```diff
@@ -0,0 +1,982 @@
+/*********************                                                        */
+/*! \file arith_proof.cpp
+** \verbatim
+** Original author: Liana Hadarean
+** Major contributors: none
+** Minor contributors (to current version): none
+** This file is part of the CVC4 project.
+** Copyright (c) 2009-2014  New York University and The University of Iowa
+** See the file COPYING in the top-level source directory for licensing
+** information.\endverbatim
+**
+** \brief [[ Add one-line brief description here ]]
+**
+** [[ Add lengthier description here ]]
+** \todo document this file
+**/
+
+#include "proof/theory_proof.h"
+#include "proof/proof_manager.h"
+#include "proof/arith_proof.h"
+#include "theory/arith/theory_arith.h"
+#include <stack>
+
+using namespace CVC4;
```

Comment:
Wrap code in namespaces instead of using using. i.e. namespace CVC4 { ... }. Not having this can cause inadvertent one-definition-rule violations in the future. So if eqNode appears in another file with the same signature in the global namespace, this would be an example. (Not sure if inline/static modifies this sufficiently.) We have had nasty bugs due to this before.